src/styles/colours.js

```diff
@@ -0,0 +1,78 @@
+// https://chir.ag/projects/name-that-color/#6195ED
+const colorDefinition = {
+  // Note: Used to be defined in foundation _settings.scss
+  // should be removed
+  primaryColor: '#1779ba',
+  // Primary palette
+  colourSapphireBlue: '#014371',
+  colourSeaBlue: '#00639a',
+  colourVividCerulean: '#00a6d5',
+  colourMediumTurquoise: '#46d6fa',
+  colourGainsborough: '#d2dce3',
+  // Grey palette
+  colourYankeesBlue: '#161d39',
+  colourIndependence: '#4e5a71',
+  colourWeldonBlue: '#8194a1',
+  colourPastelBlue: '#abc7d6',
+  colourPlatinum: '#e4e8eb',
+  colourSkyWhite: '#fbfeff',
+  // Curation
+  colourBuddhaGold: '#c39b00',
+  colourSilver: '#c0c0c0',
+  // Namespaces
+  colourJaffa: '#f2994c',
+  colourDeYork: '#88c19d',
+  colourTerracotta: '#e56358',
+  colourAmethyst: '#A748BD',
+  colourCalypso: '#357B92',
+  colourCerulean: '#00A6D5',
+  colourTurmeric: '#B8CE48',
+  colourSunglow: '#ffcc33',
+  colourFroly: '#f36968',
+  colourSalem: '#108f3b',
+  colourMalibu: '#79cbf8',
+  colourCoyoteBrown: '#966336',
+  colourOuterSpace: '#374343',
```

Comment:
Is hungarian notation necessary here? e.g. prefixing everything with `colour`? This is generally not recommended, so it will be good if we can get rid of it.

Reply:
About the use of Hungarian notation for these variables: as these are used globally as css variables, I think it's important to keep the "colour" prefix. `$colour-independence` is more meaningful than `$independence`.